ksp_plugin/plugin.cpp

```diff
@@ -1300,10 +1311,33 @@ std::uint64_t Plugin::FingerprintCelestialJacobiKeplerian(
   return Fingerprint2011(serialized.c_str(), serialized.size());
 }
 
+void Plugin::AddPart(not_null<Vessel*> const vessel,
+                     PartId const part_id,
+                     Mass const mass,
+                     DegreesOfFreedom<Barycentric> const& degrees_of_freedom) {
+  std::map<PartId, not_null<Vessel*>>::iterator it;
+  bool emplaced;
+  std::tie(it, emplaced) = part_id_to_vessel_.emplace(part_id, vessel);
+  CHECK(emplaced);
+  auto deletion_callback = [ it, &map = part_id_to_vessel_ ] {
```

Comment:
Done; clang-format does that when capturing an expression by reference somehow O_o